exercises/bob/introduction.md

```diff
@@ -0,0 +1,18 @@
+# Introduction
+
+Bob is a teenager.
+He's not very smart, but he likes to think that he's very cool.
+He definitely doesn't get excited about things.
+That wouldn't be cool.
+
+When people talk to him, his responses are pretty limited.
+
+If you ask him a question, such as "How are you?", Bob answers 'Sure.'
```

Comment:
Was the single/double quote inconsistency intended?

```suggestion
If you ask him a question, such as "How are you?", Bob answers "Sure."
```

Reply:
Not intended! Nice catch, thank you.